manticore/native/cpu/abstractcpu.py

```diff
@@ -563,6 +575,18 @@ def __setattr__(self, name, value):
         except AttributeError:
             object.__setattr__(self, name, value)
 
+    def emulate_until(self, target):
+        '''
+        Tells the CPU to set up a concrete unicorn emulator and use it to execute instructions
+        until target is reached.
+
+        :param int target: Where Unicorn should hand control back to Manticore. Set to 0 for all instructions.
+        '''
+        self._concrete = True
+        self._break_unicorn_at = target
+        if hasattr(self, 'emu'):
+            self.emu._stop_at = target
```

Comment:
Can we do this just in `concrete_emulate`?

Reply:
This is the function that the user interfaces with in order to invoke Unicorn mode, so I think it makes sense to have it as a separate function. I'll post an example script in a comment on the PR.